packages/sync-repo-settings/src/required-checks.json

```diff
@@ -42,8 +42,17 @@
         "requiresCodeOwnerReviews": true,
         "requiresStrictStatusChecks": true,
         "requiredStatusCheckContexts": [
-          "Kokoro",
```

Comment:
We probably need to wait until system tests are migrated before removing this, unless there are few enough repos that it'd make sense to have a custom `sync-repo-settings.yaml` in those repos temporarily?